commitizen/commands/check.py

```diff
@@ -50,14 +50,17 @@ def __call__(self):
             raise NoCommitsFoundError(f"No commit found with range: '{self.rev_range}'")
 
         pattern = self.cz.schema_pattern()
+        ill_formated_commits = []
```

Comment:
Maybe we could try list comprehension here. I'd suggest not adding `f"commit: {...}\n"` here to keep the variable simple and handle the representation below.

```python
ill_formated_commits = [
    commit_msg
    for commit_msg in commit_msgs
    if not Check.validate_commit_message(commit_msg, pattern)
]
```

Reply:
Thank you for your review. I've fixed it.